hazelcast/include/hazelcast/client/protocol/ClientMessage.h

```diff
@@ -392,6 +398,14 @@ class HAZELCAST_API ClientMessage
             rd_ptr(INT32_SIZE));
     }
 
+    template<typename T>
+    typename std::enable_if<std::is_same<T, float>::value, T>::type inline get()
+    {
+        return boost::endian::
+          endian_load<float, 4, boost::endian::order::little>(
```

Comment:
This seems to use only little endian encoding, but the user can change the endianness in the config.

Reply:
The endianness in the client protocol is not configurable. See protocol doc at [here](https://hazelcast.atlassian.net/wiki/spaces/HZC/pages/31948836/Hazelcast+Open+Binary+Client+Protocol)